README.md

```diff
@@ -144,6 +144,18 @@ $ sw-precache --root=dist --static-file-globs='dist/**/*.html'
 to your shell (such as the `*` characters in the sample command line above,
 for example).
 
+Finally, there's support for storing a complex configuration in an external
+JSON file, using `--config <file>`. Any of the options from the file can be
+overridden via a command-line flag. For example,
+
```

Comment:
Perhaps include a very basic snippet of what the config file would look like. I just find it easier to grok what it means with an example.